components/engine_skiplist/src/snapshot.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+use crate::db_vector::SkiplistDBVector;
+use crate::engine::{SkiplistEngine, SkiplistEngineIterator};
+use crossbeam_skiplist::map::SkipMap;
+use engine_traits::{
+    CFNamesExt, IterOptions, Iterable, Iterator, Peekable, ReadOptions, Result, SeekKey, Snapshot,
+};
+use std::ops::Deref;
+use std::sync::Arc;
+
+#[derive(Clone, Debug)]
+pub struct SkiplistSnapshot {
```

Comment:
So this in-memory-engine does not support Snapshot-Read, does it?

Reply:
Yes, so it can only be used in the stand-alone deployment without region migration.